apps/src/applab/DesignToggleRow.jsx

```diff
@@ -141,29 +147,36 @@ module.exports = React.createClass({
     }
 
     return (
-      <div className={this.props.hideToggle ? 'rightalign-contents' : 'justify-contents'}>
-        <button
-            id='codeModeButton'
-            style={$.extend({}, codeButtonStyle,
-                this.state.mode === Mode.CODE ? active : inactive,
-                this.props.hideToggle ? hidden : null)}
-            className='no-outline'
-            onClick={this.handleSetMode.bind(this, Mode.CODE)}>
-          {msg.codeMode()}
-        </button>
-        <button
-            id='designModeButton'
-            style={$.extend({}, designButtonStyle,
-                this.state.mode === Mode.DESIGN ? active : inactive,
-                this.props.hideToggle ? hidden : null)}
-            className='no-outline'
-            onClick={this.handleSetMode.bind(this, Mode.DESIGN)}>
-          {msg.designMode()}
-        </button>
-        {' ' /* Needed for "text-align: justify;" to work. */ }
-        {selectDropdown}
-        {showDataButton}
-      </div>
+      <table style={{width: '100%'}}>
+        <tbody>
+          <tr>
+            <td style={{width: '120px'}}>
+              <button
+                  id='codeModeButton'
+                  style={$.extend({}, codeButtonStyle,
+                      this.state.mode === Mode.CODE ? active : inactive,
+                      this.props.hideToggle ? hidden : null)}
```

Comment:
should we extract these semi-complicated styles into their own variables, similar to dropdownStyle, showDataButtonStyle, etc.